src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -214,6 +232,35 @@ def __init__(self, streams, stage, area, overlap, settings_obs=None, log_path=No
         self._weaver = weaver
         self._focus_plane = {}
 
+    def _normalize_region_to_polygon(
+            self,
+            region: Union[Tuple[float, float, float, float], List[Tuple[float, float]]]
+        ) -> Polygon:
+        """
+        Normalize the region input into a polygon.
```

Comment:
Can you extend this comment a bit. Looking it at the code it "normalizes" (the region if it is a rectangle and doesn't do anything if it's a polygon.

Reply:
I will rename the function to `_convert_region_to_polygon`